src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/study/StudyReportWeek.java

```diff
@@ -0,0 +1,103 @@
+package org.sagebionetworks.bridge.models.schedules2.adherence.study;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.ElementCollection;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.hibernate.WeeklyAdherenceReportRowListConverter;
+import org.sagebionetworks.bridge.models.schedules2.adherence.eventstream.EventStreamDay;
+import org.sagebionetworks.bridge.models.schedules2.adherence.weekly.WeeklyAdherenceReportRow;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonPropertyOrder({ "startDate", "endDate", "week", "adherencePercent", "rows", "byDayEntries", "type" })
+public class StudyReportWeek {
+    
+    private int week;
+    private LocalDate startDate;
+    private LocalDate endDate;
+    private Integer adherencePercent;
+    private Map<Integer, List<EventStreamDay>> byDayEntries;
+    @CollectionTable(name = "WeeklyAdherenceReportLabels", joinColumns = {
+        @JoinColumn(name = "appId"), @JoinColumn(name = "studyId"), @JoinColumn(name = "userId")})
+    @Column(name = "label")
+    @ElementCollection(fetch = FetchType.EAGER)
+    private Set<String> searchableLabels;
+    @Convert(converter = WeeklyAdherenceReportRowListConverter.class)
+    private List<WeeklyAdherenceReportRow> rows;
```

Comment:
These Hibernate annotations can be removed, this object is not directly persisted, these are all already in Weekly AdherenceReport (which is how this information is persisted).

Reply:
Removed

Reply:
Removed